.typos.toml

```diff
@@ -15,5 +15,30 @@ extend-ignore-re = [
 [default.extend-identifiers]
 "Viet_Nam" = "Viet_Nam"
 "Viet_Nam_Socialist_Republic_of" = "Viet_Nam_Socialist_Republic_of"
+"GOST" = "GOST"
```

Comment:
@PrinsFrank Biog NO! Please see my PR #235

Reply:
Yeah, that's way better, sorry about that. ;)